image/rhel/konflux.Dockerfile

```diff
@@ -63,6 +63,8 @@ FROM registry.access.redhat.com/ubi8/nodejs-18:latest AS ui-builder
 
 WORKDIR /go/src/github.com/stackrox/rox/app
 
+USER root
```

Comment:
https://console.redhat.com/preview/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/main-build-n6mbp

-> permissions?

Reply:
```
[5/6] STEP 2/7: WORKDIR /go/src/github.com/stackrox/rox/app
[5/6] STEP 3/7: COPY . .
[5/6] STEP 4/7: RUN . /cachi2/cachi2.env &&     cd .konflux/bootstrap-yarn &&     npm ci --no-audit --no-fund
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /go/src/github.com/stackrox/rox/app/.konflux/bootstrap-yarn/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, mkdir '/go/src/github.com/stackrox/rox/app/.konflux/bootstrap-yarn/node_modules'
npm ERR!  [Error: EACCES: permission denied, mkdir '/go/src/github.com/stackrox/rox/app/.konflux/bootstrap-yarn/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/go/src/github.com/stackrox/rox/app/.konflux/bootstrap-yarn/node_modules'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in: /opt/app-root/src/.npm/_logs/2024-06-28T06_57_19_132Z-debug-0.log
subprocess exited with status 243
subprocess exited with status 243
Error: building at STEP "RUN . /cachi2/cachi2.env &&     cd .konflux/bootstrap-yarn &&     npm ci --no-audit --no-fund": exit status 243
```

Reply:
I wouldn't worry about this much. Although, as a suggestion, instead of introducing `USER root`, you can change the below line from `COPY . .` to `COPY --chown=default . .`. `default` is the name of ootb user in `ubi8/nodejs-18` image.

Reply:
Done in 79e50b40da3974b72b69683049168bcc51251d5a